server/proto/sqlflow.proto

```diff
@@ -19,13 +19,19 @@ service SQLFlow {
     rpc Run (Request) returns (stream Response);
 }
 
+message Session {
```

Comment:
Is it right to name this struct `Session`? Should we rename it into `Cookies`?

According to https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies

> An HTTP cookie (web cookie, browser cookie) is a small piece of data that a server sends to the user's web browser. The browser may store it and send it back with the next request to the same server. 

It sounds similar to what this struct is meant to.

Reply:
Seems `session` is more appropriate: 
1. a cookie is the concept of HTTP proto, a session is the Broader concept, according to https://en.wikipedia.org/wiki/Session_(computer_science)
     > a session is a temporary and interactive information interchange between two or more communicating devices, or between a computer and user

1. We also want to store the SQL context on the server side in the session struct in the future,  and the cookie is stored on the browser, according to the design: https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/auth_design.md#session 

1. This PR is a crude implement of Session, hasn't included saving the context on the server side, because of: 
    >  the Hive driver and ODPS driver don't maintain long connections on each DB object

    according to the comment: https://github.com/sql-machine-learning/sqlflow/pull/530#discussion_r296859071 , but I think we need to implement this in the future.